docs/content/any/guides/policies/index.md

```diff
@@ -9,96 +9,38 @@ aliases:
-policies may be expressed.
-
-## Application Setup
-
-To use Oso for authorization, your application must first:
-
-1. Load the Oso library: `from oso import Oso`
-2. Create an {{% apiDeepLink class="Oso" /%}} instance: `oso = Oso()`
-3. Load a policy: `oso.load_file(policy)`.
-
-Then, at authorization time, you can call:
+# Write Custom Policies
 
-```python
-oso.is_allowed(actor, action, resource)
-```
+In Oso, policies are where authorization logic is stored. Policy files are
+separate from your application code, and are written in a declarative language
+called Polar (file extension `.polar`).
+
+Oso provides policy building blocks for common authorization models like [role-based
+access control](/guides/rbac), [attribute-based access control](/guides/attributes), and
+[resource hierarchies](/guides/hierarchies).
 
-This returns a boolean indicating whether the given `actor`
-is authorized to perform `action` on `resource` according to
-the current policy.
+This guide covers how to extend those building blocks by writing custom
+policies in Oso's policy language, Polar.
+You will learn the basic structure and operations of Polar rules, and how to
+write rules that refer to application classes, instances, and fields.
 
-The arguments to `oso.is_allowed` may be arbitrary Python
-objects. We'll often use strings for examples, but in your
-application they could be ORM model objects, URIs, numbers, etc.
+## What is a policy?
+
+In Oso, policies are where authorization logic is stored. Policy files are
+separate from your application code, and are written in a declarative language
+called Polar (file extension `.polar`).
+
+<!-- TODO: update enforcmeent link once docs are ported -->
+Your application can query a policy using the [Enforcement APIs]({{< ref href="/guides/enforcement" lang="python" >}}) to get authorization decisions.
+The most basic component of a policy is a **rule**, which describes a logical condition that must be true in order for a query to succeed.
 
 ## Rules
 
-The way you control whether `oso.is_allowed` returns `True`
-or `False` is to define and load Polar rules that **match**
-only the desired set of `actor`, `action`, and `resource`
-arguments. This matching may involve logical connectives,
-type checks, equality checks, variable bindings, field lookups,
-method calls, arithmetic, comparisons, etc. See the
-[Polar Syntax guide](polar-syntax) for a complete list
-of available operators.
+The way you control whether an authorization query succeeds is to define and
+load rules that **match** certain queries. This matching may involve
+logical connectives, type checks, equality checks, variable bindings, field
+lookups, method calls, arithmetic, comparisons, etc. See the [Polar Syntax
+guide](polar-syntax) for a complete list of available operators.
 
 Let's start with the basic syntax. A **rule definition** in Polar
 has a **name**, a **parameter list**, and an optional **body**.
```

Comment:
note that we used 'arguments' instead of 'parameters' in other guides I reviewed today — I prefer 'parameters'

Reply:
follow up for consistency